pipeline/lib/pipeline.ml

```diff
@@ -150,8 +154,13 @@ let docker_make_bench ~run_args ~repository image =
         "opam exec -- make bench";
       ]
 
-let pipeline ~conninfo ~run_args ~repository =
+let is_multicore repository =
+  List.mem (Repository.info repository) [ "art-w/ocaml"; "local/local" ]
```

Comment:
Could we do something that is more parametrized here? :D Perhaps passing the repositories where you can have more than one CPU as a list? :)